phoenix-shared/src/commonMain/kotlin/fr.acinq.phoenix/controllers/payments/Receive.kt

```diff
@@ -17,7 +17,11 @@ object Receive {
     }
 
     sealed class Intent : MVI.Intent() {
-        data class Ask(val amount: MilliSatoshi?, val desc: String?) : Intent()
+        data class Ask(
+            val amount: MilliSatoshi?,
+            val desc: String?,
+            val expiryDays: Long = 7
```

Comment:
Nitpick: even though this expiry can only be at least 7 days, I'd still make this expiry field a duration in seconds so that the UI has more flexibility.

Reply:
Yup, I agree. That's more flexible.

Reply:
Done in cc1aaf7